operations/model_test.go

```diff
@@ -243,3 +249,59 @@ func TestLoadWorkingChangesFromFile(t *testing.T) {
 
 	assert.False(conf.UncommittedChanges)
 }
+
+func TestShouldGenerateJWT(t *testing.T) {
+	ctx := context.Background()
```

Comment:
nit: in the newer Go versions, we should use `t.Context()` rather than `context.Background()` in tests to ensure that test context cancels when the test exits.